integration/tests/conftest.py

```diff
@@ -394,6 +394,18 @@ def storage_contract(web3_client, class_account) -> tp.Any:
     yield contract
 
 
+@pytest.fixture(scope="class")
+def storage_contract_with_deploy_tx(web3_client, class_account) -> tp.Any:
```

Comment:
1) we have storage_contract fixture  with the same contract in this file
2) why do you need to return contract_deploy_tx? if you need to get contract address just use contract.address

Reply:
I need contract deploy block for two of my tests, I can get it only from deploy tx. storage_contract fixture is used in many tests and it is not convenient to use it as a tuple in all these tests